recipes/r-bnlearn/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = '4.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bnlearn
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: bnlearn_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/bnlearn_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/bnlearn/bnlearn_{{ version }}.tar.gz
+
+  sha256: 78bb1d45977801ffa64f04f4319fa6e1fe9b5268f42b197d33aac074dc46f787
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('bnlearn')"  # [not win]
+    - "\"%R%\" -e \"library('bnlearn')\""  # [win]
+
+about:
+  home: http
```

Comment:
this URL is in most cases not there, this is very unfortunate and we already wasted a lot of CI time :(